# request
string inputString

---
# response 
string newString
